content/03.methods.md

```diff
@@ -5,8 +5,59 @@ Talk about dataset - Pubtator
 Talk about preprocessing Pubtator
 Talk about hand annotations for each realtion
 
-## Label Functions
-describe what a label function is and how many we created for each relation
+## Label Functions for Annotating Sentences
+We used simple pythonic functions called label functions to quickly annotate our candidates sentences.
+These functions are designed to emit: a positive label (1), a negative label (-1) 
+or abstain from emitting a label (0).
+We constructed label functions in the following three categories (Databases, Text Patterns and Domain Heuristics).
+These categories are described below using the following candidate sentence as an example: \"<span style="color:blue">PTK6</span> may be a novel therapeutic target for <span style="color:brown">pancreatic cancer</span>."
+
+**Databases**: This category of label functions are designed to utilize publicly available databases to determine if a co-mention pair has been previously established.
+It emits a positive label if the co-mention pair is located within a database, otherwise the given label function will abstain.
+If the label function abstains, another label function will emit a negative label.
+
+$$ \Lambda_{DB}(\color{#875442}{D}, \color{#02b3e4}{G}) = 
+\begin{cases}
+ 1 & (\color{#875442}{D}, \color{#02b3e4}{G}) \in DB \\
+0 & otherwise \\
+\end{cases} $$
+
+$$ \Lambda_{\neg DB}(\color{#875442}{D}, \color{#02b3e4}{G}) = 
+\begin{cases}
+ -1 & (\color{#875442}{D}, \color{#02b3e4}{G}) \notin DB \\
+0 & otherwise \\
+\end{cases} $$
+
+**Text Patterns**: This category of label functions are designed to provide contextual information such as keywords, number of words between two mentions and rules on grammatical structure of a sentence.
+The polarity of these functions can vary between positive and negative depending on the situation.
+
+$$ \Lambda_{TP}(\color{#875442}{D}, \color{#02b3e4}{G}) = 
+\begin{cases}
+ 1 & "target" \in Candidate \> Sentence \\
+ 0 & otherwise \\
+\end{cases} $$
+
+**Domain Heuristics**: This category of label functions is designed to incorporate domain expert knowledge and heuristic rules to annotation sentences.
+We used the findings of Percha et al.[@doi:10.1093/bioinformatics/bty114] to generate label functions of this category.
+If a sentence was highlighted in their study, our label functions emitted a positive label.
+$$
+\Lambda_{DH}(\color{#875442}{D}, \color{#02b3e4}{G}) = \begin{cases}
+    1 & Candidate \> Sentence \in Experimental \> Results\\
```

Comment:
Is `Experimental Results` a specific thing in their work or your term for it? Make sure your terminology matches what they use in their paper :)